config/default/manager_auth_proxy_patch.yaml

```diff
@@ -54,4 +54,4 @@ spec:
         - "--leader-elect"
         - "--wds-name={{.Values.ControlPlaneName}}"
         - "--api-groups={{.Values.APIGroups}}"
-        - -v=2
+        - -v=5
```

Comment:
I made this change for debugging; we probably do not want to run this high normally.